include/mitsuba/render/shape.h

```diff
@@ -148,8 +151,11 @@ class MI_EXPORT_LIB Shape : public Object {
      *
      * \param ray
      *     The ray to be tested for an intersection
+     * 
+     * \param prim_index
+     *     Index of the primitive to be intersected
```

Comment:
Would be great to indicate what the default value `prim_index=0` do. Is this a special case?

Reply:
Good point. I've added in the `sdf` branch a bit more info on the default behaviour and how for shape's that don't have multiple primitives this index is ignored. Hope that's fine.